docs/contributing/publishing-an-existing-project-to-github.md

```diff
@@ -0,0 +1,11 @@
+# Publishing an existing project to GitHub
+
+1. Open a solution in Visual Studio.
+2. If solution is not already initialized as a git repository, select **Add to Source Control** from the **File** menu. TODO: screenshot
+3. Open **Team Explorer**. TODO: screenshot
+4. Click the **Sync** button in Team Explorer. TODO: screenshot
+5. Click the **Publish to GitHub** button. TODO: screenshot
```

Comment:
> Click the **Publish to GitHub** button.

We don't explicitly call out "button" or "link" or "menu", we just refer to the label in bold: https://manual-of-style.githubapp.com/typography/#ui-elements